src/main/java/hudson/plugins/matrix_configuration_parameter/DefaultMatrixCombinationsParameterValue.java

```diff
@@ -91,6 +94,11 @@ public boolean equals(Object obj) {
     }
     
     @Override
+    public int hashCode() {
+        return (combinationFilter != null) ? combinationFilter.hashCode() : 0;
+    }
+    
+    @Override

```

Comment:
<BugInstance type='HE_EQUALS_NO_HASHCODE' priority='Normal' category='BAD_PRACTICE' message='hudson.plugins.matrix_configuration_parameter.DefaultMatrixCombinationsParameterValue defines equals but not hashCode' lineNumber='78'/>